### PR TITLE
#37 icons

### PR DESCRIPTION
**Pull request by luisferyimi19**
Changed views, models, controllers and listing in dashboard.

### CODE REVIEW

File: app/models/brand.rb [hunk 1]
```diff
@@ -3,6 +3,18 @@ class Brand < ApplicationRecord
   has_many :products
   validates :name, presence: true, uniqueness: true
   paginates_per 10
-
   scope :ci_find, lambda { |attribute, value| where("lower(#{attribute}) = ?", value.downcase).first}
+
+  def save
```
Comment (JMente) [inline app/models/brand.rb]: Fix indentation please

File: app/models/brand.rb [hunk 2]
```diff
@@ -3,4 +3,17 @@ class Brand < ApplicationRecord
...
+    if Brand.exists?(name: name)
+      errors.add(:name, :duplicated)
+      false
+    else
+      super
+    end
+  else
+    super
+  end
+end
```
Comment (Chofo2003) [inline app/models/brand.rb]: Nested if's.

File: app/models/client.rb [hunk 3]
```diff
@@ -3,4 +3,19 @@ class Client < ApplicationRecord
...
+      if Client.exists?(nit: nit)
+        errors.add(:nit, :duplicated)
+        false
+      else
+        super
+      end
+    else
+      super
+    end
+  end
```
Comment (Chofo2003) [inline app/models/client.rb]: Nested if's

File: app/models/service.rb [hunk 4]
```diff
@@ -1,14 +1,23 @@
...
+        false
+      else
+        if new_record?
+          self.actual_price = creation_price
+        end
+        super
+      end
+    else
+      super
     end
```
Comment (Chofo2003) [inline app/models/service.rb]: Nested if's

File: app/models/brand.rb (hunk 1 above)
Comment (luisferyimi19) [reply app/models/brand.rb]: Made it in the commit "requested changes"

File: app/models/brand.rb (hunk 2 above)
Comment (luisferyimi19) [reply app/models/brand.rb]: Made it in the commit "requested changes"

File: app/models/client.rb (hunk 3 above)
Comment (luisferyimi19) [reply app/models/client.rb]: Made it in the commit "requested changes"

File: app/models/service.rb (hunk 4 above)
Comment (luisferyimi19) [reply app/models/service.rb]: Made it in the commit "requested changes"

**Pull request merged**